test/CreatorToken.t.sol

```diff
@@ -484,3 +497,51 @@ contract Pausing is CreatorTokenTest {
     vm.stopPrank();
   }
 }
+
+contract bondingCurveTests is CreatorTokenTest {
+  function test_BuyPriceIsCorrect(address _buyer) public {
+    vm.assume(
+      _buyer != address(0) && _buyer != address(creatorToken) && _buyer != creator
+        && _buyer != admin
+    );
+    uint256 buyNTokens = 5;
+
+    for (uint256 i = 0; i < buyNTokens; i++) {
+      uint256 originalBuyerBalance = creatorToken.balanceOf(_buyer);
+      (uint256 _tokenPrice, uint256 _creatorFee, uint256 _adminFee) = creatorToken.nextBuyPrice();
+      uint256 _totalPrice = _tokenPrice + _creatorFee + _adminFee;
+      require(_tokenPrice == bondingCurve.priceForTokenNumber(creatorToken.totalSupply() + 1));
```

Comment:
The core thing we're trying to test here is that the price quoted (and then ultimately charged) by the creator token contracts matches the result from the bonding curve contract at the same token number N. This require statement does that sufficiently, but it should be an `assert` statement because that is what we're actually testing as opposed to an invariant of the test suite, where a require would be appropriate. Does the distinction make sense?

Reply:
That does, I was just conflicted about still being in the `1. setUp` stage of the test where `sellAToken` is the `2. execution` and asserting token balance is the `3. check`. But assert makes sense to since as you said it's what we're trying to test for here.

Reply:
Updated in same commit as the one below: https://github.com/showtime-xyz/creator-tokens/pull/14/commits/6a0ad6858e103fa6da6384492f59571f85783ba2